.github/workflows/workflow.yml

```diff
@@ -166,4 +166,4 @@ jobs:
         run: |
           export CXXFLAGS="-LDK -LND"
           make
-          ASAN_OPTIONS=detect_leaks=1 FUZZ=bolt7_channel_announcement ./bitcoinfuzz -runs=100
\ No newline at end of file
+          ASAN_OPTIONS=detect_leaks=0 FUZZ=bolt7_channel_announcement ./bitcoinfuzz -runs=100
```

Comment:
If you're going to disable `detect_leaks` so it's better to remove it instead of setting to 0.